source/guides/how-to-use-GitHub-to-visualise-your-work.html.md.erb

```diff
@@ -0,0 +1,14 @@
+---
+title: How to use GitHub to visualise your work
```

Comment:
I think 'visualise' feels like the wrong word to me. We use 'manage' in the sentence straight after which aligns more with how I think we use Github. It sounds a bit more involved and like Github's a useful tool, rather than an inconvenience that you need to do to make your work visible... if that makes sense!

Reply:
Amended!